src/components/RegionSelector/index.tsx

```diff
@@ -77,12 +82,11 @@ const RegionSelector: React.FC<RegionSelectorProps> = ({
                         }) ?? selectedRegion.english_name
                       : isUserSetting && selectedRegion?.iso_3166_1 !== 'all'
                       ? intl.formatMessage(messages.regionServerDefault, {
-                          applicationTitle: currentSettings.applicationTitle,
                           region: currentSettings.region
                             ? intl.formatDisplayName(currentSettings.region, {
                                 type: 'region',
                                 fallback: 'none',
-                              }) ?? currentSettings.region
+                              })
```

Comment:
You still need the `currentSettings.region` for the fallback when formatDisplayName cant find the localization of the region.

Reply:
Actually its probably easier to set `fallback` in this case to `code`. Since we would be falling back to the code anyways in this rare case.

Reply:
Actually, on iOS this is an even bigger problem because the intl support for display names is really bad. I think we need to actually search through the tmdb regions for a matching code and use the `english_name` provided by them.